templates/components/header.jsx

```diff
@@ -4,8 +4,13 @@ var Header = React.createClass({
   render: function() {
     return (
       <div className="navbar">
-        <img src="/assets/img/logo.png" alt="logo" className="logo" />
-        <div className="sign-up">Need an account? <a href="" className="underline">Sign up</a></div>
+        {
+          !!this.props.wordmark ?
+            <img src="/assets/img/webmaker-horizontal.svg" alt="Mozilla Webmaker" className="wordmark" />
+            :
+            <img src="/assets/img/logo.png" alt="Mozilla Webmaker" className="logo" />
+        }
+        <div className="redirect">{this.props.redirectText} <a href={this.props.redirectURL} className="underline">{this.props.redirectLabel}</a></div>
```

Comment:
Can you update href to use this instead? 

```
<Link to={this.props.redirectPage} className="underline">{this.props.redirectLabel}</Link>
```

instead of 

```
<a href={this.props.redirectURL} className="underline">{this.props.redirectLabel}</a>
```

This way it will not do an actual page load redirect.